### ISSUE TITLE
using bazel 0.19.2 to build beacon-chain failed

### ISSUE THREAD

**Issue by yaanhyy** [prysmaticlabs/prysm#1022]
When I run command,get below hint.My GOPTH=/home/yaanhyy/go_work,and have the package:
```
$ bazel build //beacon-chain:beacon-chain
Starting local Bazel server and connecting to it...
ERROR: Analysis of target '//beacon-chain:beacon-chain' failed; build aborted: no such package '@com_github_ethereum_go_ethereum//ethclient': BUILD file not found on package path
INFO: Elapsed time: 6.669s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (27 packages loaded, 6113 targets \
configured)

```

go-ethereum package：
```
yaanhyy@yaanhyy-System-Product-Name:~/go_work/src/github.com/prysmaticlabs/prysm$ ll ~/go_work/src/github.com/ethereum/go-ethereum/ethclient/
total 36
drwxrwxr-x  2 yaanhyy yaanhyy  4096 12月  1 21:18 ./
drwxrwxr-x 38 yaanhyy yaanhyy  4096 12月  1 21:18 ../
-rw-rw-r--  1 yaanhyy yaanhyy 18973 12月  1 21:18 ethclient.go
-rw-rw-r--  1 yaanhyy yaanhyy  4027 12月  1 21:18 ethclient_test.go
-rw-rw-r--  1 yaanhyy yaanhyy  2097 12月  1 21:18 signer.go
```

**Comment by yaanhyy** [comment]
Now， I  add proxy fixed the problem.But meet new issue, how can I fixed it:
```
ERROR: /home/yaanhyy/.cache/bazel/_bazel_yaanhyy/3e6dcb143d9695706229cc96b785f9db/external/io_bazel_rules_go/proto/wkt/BUILD.bazel:3:1: error loading package '@com_google_protobuf//': Extension file not found. Unable to load file '@bazel_skylib//lib:versions.bzl': file doesn't exist or isn't a file and referenced by '@io_bazel_rules_go//proto/wkt:timestamp_go_proto'
ERROR: Analysis of target '//beacon-chain:beacon-chain' failed; build aborted: error loading package '@com_google_protobuf//': Extension file not found. Unable to load file '@bazel_skylib//lib:versions.bzl': file doesn't exist or isn't a file
INFO: Elapsed time: 0.348s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 5 targets conf\
igured)
    currently loading: @com_google_protobuf//

```

**Comment by prestonvanloon** [comment]
From which commit are you building? This was fixed in #1016 and #1020.

You may need to do a full clean and rebuild.
```
bazel clean —expunge
bazel build //beacon-chain
```

**Comment by prestonvanloon** [comment]
This builds on master now with a clean build.

Here's the version I tested with:
```
$ bazel version
Build label: 0.19.2
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Mon Nov 19 16:25:09 2018 (1542644709)
Build timestamp: 1542644709
Build timestamp as int: 1542644709
```
